### ISSUE TITLE
Can you share your pre-trained model on RaFD dataset?

### ISSUE THREAD

**Issue by realliujiaxu** [yunjey/StarGAN#131]
I'm citing your paper as a baseline method. Could you give me your pre-trained model on RaFD dataset? Thanks a lot.

**Comment by marcopoli** [comment]
I agree I downloaded the RaFD dataset but training a model is very slow. 1 minute and 24 seconds every 10 iterations on a tesla V100. I will end my 200000 iterations in 414 days.

**Comment by Vanleya** [comment]
have u solved this problem? thans
